src/terminal_ui.cc

```diff
@@ -181,7 +181,22 @@ void TerminalUI::Screen::set_face(const Face& face)
 {
     static constexpr int fg_table[]{ 39, 30, 31, 32, 33, 34, 35, 36, 37, 90, 91, 92, 93, 94, 95, 96, 97 };
     static constexpr int bg_table[]{ 49, 40, 41, 42, 43, 44, 45, 46, 47, 100, 101, 102, 103, 104, 105, 106, 107 };
-    static constexpr int attr_table[]{ 0, 4, 7, 5, 1, 2, 3 };
```

Comment:
while the new code is a bit more explicit, I kind of like the simplicity of having a onliner table, which should be possible again if you change Strikethrough as requested above.